composeApp/src/commonMain/kotlin/org/ooni/probe/ui/dashboard/DashboardScreen.kt

```diff
@@ -2,185 +2,180 @@ package org.ooni.probe.ui.dashboard
+import ooniprobe.composeapp.generated.resources.TestResults_Overview_Websites_Blocked
+import ooniprobe.composeapp.generated.resources.TestResults_Overview_Websites_Tested
 import ooniprobe.composeapp.generated.resources.app_name
 import ooniprobe.composeapp.generated.resources.dashboard_arc
-import ooniprobe.composeapp.generated.resources.ic_keyboard_arrow_down
-import ooniprobe.composeapp.generated.resources.ic_keyboard_arrow_up
+import ooniprobe.composeapp.generated.resources.ic_auto_run
+import ooniprobe.composeapp.generated.resources.ic_heart
+import ooniprobe.composeapp.generated.resources.ic_history
+import ooniprobe.composeapp.generated.resources.ic_measurement_anomaly
+import ooniprobe.composeapp.generated.resources.ic_measurement_failed
+import ooniprobe.composeapp.generated.resources.ic_tests
 import ooniprobe.composeapp.generated.resources.ic_warning
+import ooniprobe.composeapp.generated.resources.ic_world
 import ooniprobe.composeapp.generated.resources.logo_probe
+import org.jetbrains.compose.resources.DrawableResource
 import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.compose.resources.pluralStringResource
 import org.jetbrains.compose.resources.stringResource
 import org.jetbrains.compose.ui.tooling.preview.Preview
-import org.ooni.probe.data.models.DescriptorType
-import org.ooni.probe.data.models.DescriptorUpdateOperationState
+import org.ooni.probe.config.OrganizationConfig
+import org.ooni.probe.data.models.ArticleModel
+import org.ooni.probe.data.models.MeasurementStats
+import org.ooni.probe.data.models.Run
+import org.ooni.probe.data.models.RunBackgroundState
+import org.ooni.probe.shared.largeNumberShort
+import org.ooni.probe.ui.articles.ArticleCell
 import org.ooni.probe.ui.shared.IgnoreBatteryOptimizationDialog
 import org.ooni.probe.ui.shared.TestRunErrorMessages
-import org.ooni.probe.ui.shared.UpdateProgressStatus
 import org.ooni.probe.ui.shared.VerticalScrollbar
 import org.ooni.probe.ui.shared.isHeightCompact
+import org.ooni.probe.ui.shared.relativeDateTime
 import org.ooni.probe.ui.theme.AppTheme
+import org.ooni.probe.ui.theme.LocalCustomColors
+import org.ooni.probe.ui.theme.cardTitle
+import org.ooni.probe.ui.theme.dashboardSectionTitle
 
 @Composable
 fun DashboardScreen(
     state: DashboardViewModel.State,
     onEvent: (DashboardViewModel.Event) -> Unit,
 ) {
-    val pullRefreshState = rememberPullToRefreshState()
-    Box(
-        Modifier
-            .pullToRefresh(
-                isRefreshing = state.isRefreshing,
-                onRefresh = { onEvent(DashboardViewModel.Event.FetchUpdatedDescriptors) },
-                state = pullRefreshState,
-                enabled = state.isRefreshEnabled && state.canPullToRefresh,
-            ).background(MaterialTheme.colorScheme.background)
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .background(MaterialTheme.colorScheme.background)
```

Comment:
This seems like a missed opportunity to enable pull to refresh.

I started the application without connectivity and could not get articles till restart.